sdk/arm/resources/2020-06-01/armresources/README.md

```diff
@@ -0,0 +1,48 @@
+# Azure Resources Module for Go
+
+[![PkgGoDev](https://pkg.go.dev/badge/github.com/Azure/azure-sdk-for-go/sdk/arm/resources/2020-06-01/armresources)](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/arm/resources/2020-06-01/armresources)
+
+The `armresources` module provides operations for working with resources and resource groups.
+
+[Source code](https://github.com/Azure/azure-sdk-for-go/tree/master/sdk/arm/resources/2020-06-01/armresources)
+
+# Getting started
+
+## Install the package
+
+This project uses [Go modules](https://github.com/golang/go/wiki/Modules) for versioning and dependency management.
+
+Install the Azure Resources module:
+
+```sh
+go get -u github.com/Azure/azure-sdk-for-go/sdk/arm/resources/2020-06-01/armresources
+```
+
+## Prerequisites
+
+- an [Azure subscription](https://azure.microsoft.com/free/)
+- Go 1.13 or above
```

Comment:
Should this be go1.14 and above now?

Reply:
I was thinking about that.  All the others are Go 1.13, and all of our build tags specify 1.13 as the minimum version.  I'm inclined to leave it as 1.13 for now and if we decide we want to bump the minimum to 1.14 then we do a repo-wide sweep.

Reply:
Hmmm ok, we can discuss when to bump everything up to 1.14 later.

Reply:
Also I believe that our code will work with 1.13, so while we don't have to support I don't feel that we have to go out of our way to prevent it from working.